spec/array_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'array'
+
+describe Array do
+  context '#gusject method uses blocks' do
+    it 'sums all the members of the array' do
+      expect([2, 3, 4].gusject do |sum, el|
+        sum + el
+      end).to eq([2, 3, 4].inject { |sum, el| sum + el })
```

Comment:
I did think about it, but then I would have set it to something like 'test_array'. That symbol actually takes up more characters than the array literal. Thus decided against it. Any thoughts on that?